### DISCUSSION TITLE
2024/09/14/docker-with-vnc/

### DISCUSSION THREAD

**Discussion by giscus** [wu1015/wu1015.github.io#4] [Announcements]
# 2024/09/14/docker-with-vnc/

vnc配置前的准备启动容器 这一步比较关键，要注意启动的镜像的版本。我们这里以Ubuntu举例，如果高于22.04，就必须加入参数--security-opt seccomp=unconfined。否则会出问题。在下述启动命令中，指定使用主机所有gpu资源，命名为wu1015_test，将容器5901端口映射到本地的32789。1docker run  --gpus all  --ipc=host

https://wu1015.github.io/2024/09/14/docker-with-vnc/

**Comment by wu1015** [comment]
`测试评论`
